dask_cuda/local_cuda_cluster.py

```diff
@@ -45,11 +48,17 @@ def __init__(
             raise ValueError("Can not specify more processes than GPUs")
         if memory_limit is None:
             memory_limit = TOTAL_MEMORY / n_workers
+        if device_memory_limit is None:
+            device_memory_limit = get_device_total_memory() / n_workers
```

Comment:
There might be an error here . 

I was playing with a node  with 8 '16 GB' gpus and the memory of a single device seems to get divided here.  I guess  ``get_device_total_memory`` returns memory for a single gpu. 


```python
from dask_cuda import LocalCUDACluster
cluster = LocalCUDACluster(ip='0.0.0.0')
print("Memory = {} GB".format(float(cluster.worker_kwargs['device_memory_limit'])/float(1<<30)))

cluster = LocalCUDACluster(ip='0.0.0.0',n_workers = 1)
print("Memory = {} GB".format(float(cluster.worker_kwargs['device_memory_limit'])/float(1<<30)))
```

```python
Memory = 1.9874114990234375 GB
Memory = 15.8992919921875 GB
```

Reply:
Thanks for pointing it out @VibhuJawa. I did notice that before and even had an uncommitted fix, but I forgot to push it. Could you pull the new commit and check if it works?

Reply:
Yup. It works now. Thanks for the quick fix. 👍